docs/reference/glossary.md

```diff
@@ -12,7 +12,10 @@ DKMS
   See the dkms manpages for more information.
 
 ABI
-  Application binary interface.
+  Application Binary Interface, or ABI defines a stable interface between userspace
```

Comment:
Apologies @Jetoson , I think you may have misread this.
The suggestion was to spell it out without a hyphen, "user space", similar to how the Linux kernel docs does it.

Do submit the change and squash the changes; i'll merge it right after.

Thanks!